201-managed-application-usingmsi-for-management/metadata.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
validate this file against the schema and fix errors (see CI results)

Reply:
Fixed